api/v1/app.py

```diff
@@ -37,3 +36,6 @@ def close_connection(exception):
 
 if __name__ == '__main__':
    app.run(host="0.0.0.0", port=5000, threaded=True)
```

Comment:
Try to avoid hardcoded values! (get environment variables here because sometimes we need to switch ports)

Reply:
change made